app/views/viewTransfers.scala.html

```diff
@@ -1,62 +1,73 @@
-            Back to homepage
-        </a>
+      <h1 class="govuk-heading-l">View Transfers</h1>
+      <p class="govuk-body">View the history of all the consignments you have uploaded. You can also resume incomplete transfers or view the errors of failed transfers.</p>
+
+      <table data-module="table-row-expander" class="govuk-table govuk-table--tdr-transfers govuk-!-margin-bottom-2">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header" id="toggle">
+              <span class="govuk-visually-hidden">Expand</span></th>
+            <th scope="col" class="govuk-table__header">Reference</th>
+            <th scope="col" class="govuk-table__header">Date started</th>
+            <th scope="col" class="govuk-table__header">Date transferred</th>
+            <th scope="col" class="govuk-table__header">Status</th>
+            <th scope="col" class="govuk-table__header">Actions</th>
+            <th scope="col" class="govuk-table__header govuk-visually-hidden" id="more-details">More details</th>
+          </tr>
+        </thead>
+        @for((consignment, index) <- consignments.zipWithIndex) {
+          <tbody>
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell" rowspan="2">
+                <button class="tdr-arrow-expander" aria-expanded="false" aria-controls="details-@index">
+                  <span class="govuk-visually-hidden">More details about @consignment.reference</span>
+                </button>
+              </td>
+              <th scope="row" class="govuk-table__header">@consignment.reference</th>
+              <td class="govuk-table__cell">@consignment.dateStarted</td>
+              <td class="govuk-table__cell @{if(consignment.dateOfTransfer == "N/A") "not-applicable"}">@consignment.dateOfTransfer</td>
+              <td class="govuk-table__cell">
+                <strong class="tdr-tag tdr-tag--@consignment.statusColour">@consignment.status</strong>
+              </td>
+              <td class="govuk-table__cell">
+                <div class="tdr-link-group">
+                  <a href="@{consignment.userAction.actionUrl.format(nationalArchivesEmailAddress)}" class="govuk-link govuk-link--no-visited-state">@consignment.userAction.actionText</a>
+                </div>
+              </td>
+            </tr>
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell" headers="more-details" colspan="5" id="details-@index">
+                  @if(consignment.status == Transferred.value) {
+                    @warningMessage("""You must not delete the original records of this transfer as they are not yet preserved. You will receive an email once preservation has taken place. If you do not receive an email, contact <a href="mailto:tdr@nationalarchives.gov.uk.">tdr@nationalarchives.gov.uk</a>.""", marginValue = 0)
```

Comment:
Sorry one last thing, can you swap the email with the property?

`<a href="mailto:@Messages("nationalArchives.email")">@Messages("nationalArchives.email")</a>`

Reply:
As suggested, I've swapped the email with the email properties.